libenv/unix_iface.c

```diff
@@ -148,7 +148,7 @@ static void GetMacAddress(EvalContext *ctx, ARG_UNUSED int fd, struct ifreq *ifr
       return;
     }
 
-# if defined(SIOCGIFHWADDR) && defined(HAVE_STRUCT_IFREQ_IFR_HWADDR)
+# if defined(SIOCGIFHWADDR) && defined(HAVE_STRUCT_IFREQ_IFR_HWADDR) && !defined(__TERMUX__)
```

Comment:
this should probably be `__ANDROID__` instead of `__TERMUX__` for the eventuality that we support on-device building, which doesn't has the former rather than the later macro defined by default.